orttraining/orttraining/python/training/ortmodule/transformers/__init__.py

```diff
@@ -0,0 +1,15 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import os
+
+_all_transformers = []
```

Comment:
can we rename to graph_optimizer or something to make it clear what this does... transformer is overloaded term.